wandb/board/ui/src/components/RunEditor.js

```diff
@@ -66,13 +113,14 @@ class RunEditor extends React.Component {
               this.setState({canSubmit: false});
               this.props
                 .submit({
+                  tags: this.state.tags,
                   description: this.state.content,
                   id: this.props.bucket.id,
                 })
                 .then(res => {
-                  window.location.href = `/${this.props.model.entityName}/${this
-                    .props.model.name}/runs/${res.data.upsertBucket.bucket
-                    .name}`;
+                  window.location.href = `/${this.props.model.entityName}/${
```

Comment:
This will mean the user can't go back right? If you want to stick it in the browser history you need to use history.push which I think needs to be passed down from another component through props. Maybe there is a way to make it global though.

Reply:
Hmm I don't think setting `location.href` prevents history from being set. I think `location.replace()` does that.